### ISSUE TITLE
running parrallele processing using runcell hangs while using F9 it works

### ISSUE THREAD

**Issue by deMalmazet** [spyder-ide/spyder#12715]
### What steps will reproduce the problem?

When I run a line of code involving multiprocessing in Windows using runcell it hangs, while when using F9 it works

## Versions

* Spyder version: 4.1.3 
* Python version: 3.6.10
* Qt version: 5.12.5
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.14.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.3 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.3 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.2.1 (OK)
numpy >=1.7                    :  1.18.4 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  None (OK)
```

**Comment by dalthviz** [comment]
Hi @deMalmazet thanks for the feedback! Could you provide a code example in order to reproduce this? Any new info is greatly appreciated :)

**Comment by deMalmazet** [comment]
Hi @dalthviz,
The problem qot somewhat fixed by changing the run settings and the working directory. In working directory settings I changed it to "the current working directory",  and in console I put "Execute in a dedicated console". Finally I made sure my working directory was the same as the one from the file running.
Sometimes it still didn't work, but I can work with it for now. I'm using a complex code which calls many subfunctions. If I have time I'll try to reproduce the error with a simpler code.
Thanks
